test/test_optim.py

```diff
@@ -826,6 +827,49 @@ def fwd_bwd(optim, mod, i):
                 optimizer.step()
 
 
+    @optims(optim_db, dtypes=[torch.float32])
+    def test_save_load_state_dict(self, device, dtype, optim_info):
```

Comment:
Can this name be a little more indicative, maybe
```suggestion
    def test_save_load_equality_with_weights_only(self, device, dtype, optim_info):
```
or something of your choice